Formula/d/dvisvgm.rb

```diff
@@ -0,0 +1,58 @@
+# typed: false
+# frozen_string_literal: true
+
+class Dvisvgm < Formula
+  desc "Fast DVI to SVG converter"
+  homepage "https://dvisvgm.de"
+  url "https://github.com/mgieseki/dvisvgm/releases/download/3.5/dvisvgm-3.5.tar.gz"
+  sha256 "41ea2e10fe6bdc4ce7672519cfc2998e5c30c8b29fbcd8901915c7dac7fa494c"
+  license "GPL-3.0-or-later"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "llvm" => :build
+  depends_on "pkgconf" => :build
+  depends_on "freetype"
+  depends_on "ghostscript"
+  depends_on "texlive"
+  depends_on "woff2"
+  depends_on "zlib"
```

Comment:
This should work too
```suggestion
  uses_from_macos "zlib"
```